app/themes/user-admin/src/app/users-event-log/users-event-log.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="row">
+    <div class="col-xs-4">
+        <button class="btn btn-default btn-sm" routerLink="/user/">Create a New User</button>
+    </div>
+</div>
+<div class="row">
+    <div class="col-xs-12">
+        <br />
+    </div>
+</div>
+<div class="row">
+    <div class="col-xs-12">
+        <table class="table table-bordered table-hover"> 
+            <thead>
+                <tr>
+                    <th (click)="sort()">
```

Comment:
`sort` needs to accept a parameter for the sort column. see this as an example: https://github.com/Erdiko/user-admin/blob/master/app/themes/user-admin/src/app/user-list/user-list.component.html#L16